cds/modules/previewer/api.py

```diff
@@ -110,7 +110,11 @@ def thumbnails_uri(self):
     def subtitles(self):
         """Get video's subtitles."""
         return [
-            (f["links"]["self"], f["tags"]["language"])
+            (
+                f["links"]["self"],
+                f["tags"]["language"],
+                f["tags"].get("autogenerated", False)
```

Comment:
I am adding the following (see [here](https://gitlab.cern.ch/cds-team/production_scripts/-/merge_requests/37/diffs#cda9f76cb1a114033b250b31236334ad7dbf62b8_0_72)):
```
ObjectVersionTag.create_or_update(obj, "autogenerated", "true")
```
I think that this will not be enough, you will have to parse the value and check if the value is "true", otherwise (another string or missing) the python value can be `False`.